tests/test_bgg_companion.py

```diff
@@ -8,7 +8,13 @@
     TestGetBoardGamesData,
 )
 
+from flask_caching import Cache
+from flask import Flask
 
+test_cache = Cache(config={"CACHE_NO_NULL_WARNING": True})
```

Comment:
You might also want to specify the [`NullCache`](https://pythonhosted.org/Flask-Cache/#nullcache-null) cache type here to make sure you're not caching anything in unit tests, which could result in weird behavior

Reply:
Ahhhhh great point!